docs/Drawer.md

```diff
@@ -35,11 +35,17 @@ import { Drawer, DrawerHeader, DrawerSubheader, DrawerBody, DrawerNavGroup, Draw
 | Prop Name                       | Description                                                | Type      | Required | Default  |          
 |---------------------------------|------------------------------------------------------------|-----------|----------|----------|
 | open                            | Controls the open/closed state of the drawer               | `boolean` | yes      |          |
+| variant                         | The variant to use (see below)                             | `'permanent' | 'persistent' | 'temporary'`| no | |
 | width                           | Sets the width of the drawer (in px) when open             | `number ` | no       |          |
 | [...sharedProps](#shared-props) | Props that can be set at any level in the drawer hierarchy | -         | no       |          |
 
 </div>
 
+The `Drawer` has three `variant`s: 
+- **Permanent**: Always open, even when `open` is set to false.
+- **Persistent**: When `open` is set to false, the Drawer collapses itself as a navigation rail, and hover will make it expand temporarily; when `open` is set to true, it behaves like a permanent Drawer.
+- **Temporary**: When `open` is set to false, the drawer is hidden; when `open` is set to true, it slides in.
+The drawer variant overrides that set in variantBreakpoints.
```

Comment:
What is this line saying? If the variant property is set on the drawer, it will override any variantBreakpoint configuration in the DrawerLayout?